students/anthony_mckeever/lesson3/assignment_1/mailroom.py

```diff
@@ -0,0 +1,295 @@
+"""
+Programming In Python - Lesson 3 Assignment 1: Mailroom Part 1
+Code Poet: Anthony McKeever
+Start Date: 07/30/2019
+End Date: 08/01/2019
+"""
+
+menu_opts = ["main", "return", "stop"]
+quit_opts = ["exit", "end", "quit"]
+list_opts = ["list", "l", "ls"]
+help_opts = ["help", "h", "?"]
+
+donors_list = [("Cresenta Starchelle", [99.99, 6000.00, 10345.23, 29.99]),
+               ("Delilah Matsuka", [199.99, 299.99, 2100.00]),
+               ("Astra Matsume", [599.99]),
+               ("Kima Metoyo", [3600.00, 1200.00]),
+               ("Kayomi Matsuka", [0.01]),
+               ("Katie Starchelle", [600.00])]
+
+stay_on = True
+
+
+def main():
+    """
+    The main loop of the script.  Prompts user with a main menu.
+    """
+    print("\nStudio Starchelle Donor Appreciation System\n")
+    while stay_on:
```

Comment:
I would recommend simplifying this (especially will matter in lesosn4)

use `while True` and create a quit_program function to call `sys.exit()`

Reply:
I appreciate the feedback!  I'll definitely simplify the code for lesson 4.